preamble.tex

```diff
@@ -302,6 +302,7 @@
         else,enum,extern,float,for,goto,if,int,long,register,return,%
         short,signed,sizeof,static,struct,switch,typedef,union,unsigned,%
         void,volatile,while},
+    % henrikt-ma: How about adding some highlighting of 'size_t' as a recognized name?
```

Comment:
It doesn't seem that people normally do that, and` size_t` is "just" a typedef alias from a header and not a keyword.

Reply:
I don't know about normal people, but it seems to happen in IDEs too:
https://www.reddit.com/r/C_Programming/comments/hhf9b1/why_are_identifiers_like_size_t_and_wchar_t/

I am fully aware that this is a topic for discussion.  I just happen to belong to the category of people who find such syntax highlighting useful (in the same way we do with things like `Real`).

By the end of this discussion, let's do one of these:
- Add syntax highlighting of `size_t`.
- Remove the comment.
- Open a new issue.